src/test/resources/net/sf/jabref/importer/fileformat/BiblioscapeImporterTestUnknownType.bib

```diff
@@ -0,0 +1,6 @@
+@misc{,
+  author = {Name},
+  booktitle = {Test Title},
+  pages = {--11},
```

Comment:
See `BiblioscapeImporterTestUnknownType.txt`. The tag `--PE--` tag means "page end". Cause there is no start page tag (`--PS--`) the pages are interpreted as `--11` in bibtex. This was the purpose of this test file.